NetKAN/SVT-Windows.netkan

```diff
@@ -0,0 +1,26 @@
+{
+    "spec_version"   : "v1.4",
+    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*(?!Mac).zip$",
```

Comment:
Ok, that wasn't working, but this does (though I don't really like it)

    "$kref"          : "#/ckan/github/Galileo88/Stock-Visual-Terrain/asset_match/SVT.*[^x]\\.zip$",